test/test-utils.ts

```diff
@@ -39,6 +46,40 @@ export const createDBEntities = async (): Promise<void> => {
   });
 };
 
+export const createAdminUser = async (
+  app: INestApplication,
+): Promise<AuthTokenOutput> => {
+  const defaultAdmin: CreateUserInput = {
+    name: 'Default Admin User',
+    username: 'default-admin',
+    password: 'default-admin-password',
+    roles: [ROLE.ADMIN],
```

Comment:
@sivanpayyadakath 
I don't think we should allow any user to register themselves as an `ADMIN`. :thinking: 
Let's discuss how we can solve this issue.